OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -48,8 +48,18 @@ public class AssetBrowserLogic : ChromeLogic
 		int currentFrame;
 
 		LabelWidget errorLabelWidget;
+		LabelWidget errorLabelFilename;
 		SpriteFont errorFont;
-		bool IsErrorLabelVisible { get { return errorLabelWidget != null && errorLabelWidget.Visible; } }
+		bool IsErrorLabelVisible
```

Comment:
This isn't public so should be `isErrorLabelVisible`.
Adding `changes requested` label because of this.